ganga/GangaCore/Utility/logging/__init__.py

```diff
@@ -600,3 +600,18 @@ def log_unknown_exception():
 
     tb_logger.debug('Bare except clause triggered {0}:{1}'.format(caller.filename, caller.lineno))
     tb_logger.debug('Exception caught:', exc_info=True)
+
+def supress_logging(func):
```

Comment:
IS this function problematic in the multithreaded environment of Ganga? It could cause other messages not to come out, and if the method is entered twice, the `previousloglevel` variable might get the wrong value.

Reply:
I don't think this does what I want. Really I was trying to suppress the messages about the postprocessors running but they occur elsewhere.

I think I'll get rid of it.